yt/frontends/owls_subfind/io.py

```diff
@@ -4,9 +4,12 @@
 from yt.utilities.io_handler import BaseIOHandler
 from yt.utilities.on_demand_imports import _h5py as h5py
 
+_pos_names = ["CenterOfMass", "CentreOfMass"]
```

Comment:
I almost suggested changing this to be a tuple, and then I realized we may actually want it to be mutable.